requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+-e .
+
+requests-mock==1.8.0
+pytest==5.4.3
```

Comment:
pytest is a Python test framework that does a ton of magic and makes writing tests much easier once you get the hang of it. It does test discovery, it does magic around `assert` making it easier to debug issues, it has "fixtures" which do run-time injection of behavior and features, etc. It will be your friend if you let it.

requests-mock mocks out the requests module so you can more easily write tests that hit services on the network without hitting those services on the network.